core/tests/test_account/src/lib.rs

```diff
@@ -191,6 +195,52 @@ impl Account {
         .unwrap()
     }
 
+    pub fn get_l1_prank_tx(
+        &self,
+        execute: Execute,
+        serial_id: u64,
+        prank_address: Address,
+    ) -> Transaction {
+        let max_fee_per_gas = U256::from(0u32);
+        let gas_limit = U256::from(20_000_000);
```

Comment:
why 20 million? Why not something like 72kk (the maximal gas value for the transaction), just seems like a random constant